### PR TITLE
Trend-Micro: add the documentation to configure Workload Security

### PR DESCRIPTION
**Pull request by squioc**

### CODE REVIEW

File: docs/xdr/features/collect/integrations/endpoint/trend_micro_workload_security.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+#### Indirect forwarding
+
+If your agents should forward events over untrusted network,
+select the transport protocol `TLS` and ask your agents to send events `Via the Workload Security Manager`. 
+Generate and add a certificate to the log concentrator.
+Apply the changes and test the connection.
+
+For the first connection, the manager should display the error message "Accept Server Certificate?".
+Check the certificate and click OK to accept it. Test the connection again; it should succeed. If not, please see the "Troubleshoot event forwarding" section on the Trend-Micro documentation.
```
Comment (PierrickV) [inline docs/xdr/features/collect/integrations/endpoint/trend_micro_workload_security.md:39]:
```suggestion
Check the certificate and click OK to accept it. Test the connection again; it should succeed. If not, please see the "[Troubleshoot event forwarding](https://cloudone.trendmicro.com/docs/workload-security/event-syslog/#troubleshoot-event-forwarding)" section on the Trend-Micro documentation.
```

**Review by PierrickV — APPROVED** [review]
:+1:

File: docs/xdr/features/collect/integrations/endpoint/trend_micro_workload_security.md (hunk 1 above)
Comment (squioc) [reply docs/xdr/features/collect/integrations/endpoint/trend_micro_workload_security.md]: Good idea.
However, as now, the documentation refers to two guides; seems more difficult to redict to the "Throubleshoot" section :(

File: docs/xdr/features/collect/integrations/endpoint/trend_micro_deep_security.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+uuid: b2d961ae-0f7e-400b-879a-f97be24cc02d
+name: Trend Micro Deep Security / Workload Security
+type: intake
+
+## Overview
+
+Trend Micro Deep Security / Workload Security detect malware, vulnerabilities and any other threats.
```
Comment (PierrickV) [inline docs/xdr/features/collect/integrations/endpoint/trend_micro_deep_security.md:7]:
```suggestion
Trend Micro Deep Security / Workload Security detect malware, vulnerabilities and some other threats.
```

*any other threats* seems very magical to me.

Maybe remove this part or be more specific ?

**Pull request merged**